package.nix

```
@@ -0,0 +1,9 @@
+{ pkgs ? import <nixpkgs> {} , displayrUtils }:
+
+pkgs.rPackages.buildRPackage {
+  name = "flipU";
+  version = displayrUtils.extractRVersion (builtins.readFile ./DESCRIPTION); 
+  src = ./.;
+  description = "Utility functions used across multiple flip packages.";
+  propagatedBuildInputs = with pkgs.rPackages; [ pkgs.R R_utils ];
```

After you opened the PR I ended up responding to your Slack question about if we needed to add `pkgs.R`, and we don't as `buildRPackage` does that for us.
So we can remove it:
```suggestion
  propagatedBuildInputs = with pkgs.rPackages; [ R_utils ];
```

Separately, I'm not sure if we should be using `propagatedBuildInputs` or `nativeBuildInputs` here. It looks like upstream sets both to whatever the `depends` array is (which is from Cran/Bioc).

We could test this by building the package (as is), loading it into R, and calling a function that loads `R_utils`. If that works, we are doing it correctly (even if not perfectly efficiently).